Weaken the effect of heating on large hotboxes/puddles

TDHooligan opened the pull request:
[BALANCE]
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->

Makes temperature_reagents only heat 1 tile's worth of chems for liquids/smokes, lowering the overall heating. 

While there's some outstanding code (possibly an issue) that seems to be making the minimum increment 1 degree, it's 5x better than the surprisingly easy to achieve 500C super hotbox achievable with a couple leaves of cannabis and a lighter.


## Why's this needed? <!-- Describe why you think this should be added to the game. -->
Currently even a basic hotbox/flood becomes scalding hot in a very short time with just one candle, burning people with no mask. This slows the rate of heating down significantly.

Kinda nerfs lethal hotboxes, but odds are if you aren't wearing a mask you'd be dying to the deathweed in a deathweed cloud rather than 3 hundred degree smoke.

MetricDuck commented:
Effectively removes welder fuel spill / tank explosions which is a shame.

Did you mean to have two exposed_volume = exposed_volume =?

TDHooligan commented:
fixed. i'm fairly certain that welding fuel's ignition issue is largely mitigated by emily's previous 

> Effectively removes welder fuel spill / tank explosions which is a shame.
> 
> Did you mean to have two exposed_volume = exposed_volume =?

fixed the latter, weird git error.

the former i think is better addressed by giving flammables a proper flash point and making flammables in open containers a little less explode-y and a bit more fiery. happy to work that into this patch as Cogwerks mentioned nerfing fuel explosions properly anyway
